### ISSUE TITLE
Clean-up and replace Intellectual property chapter

### ISSUE THREAD

**Issue by egpbos** [NLeSC/guide#384]
- [x] Add a section to Best Practices.
    - [x] NLeSC-internal note: refer to the PMP, section "IP policy"
- [x] Remove the chapter: [PR together with licensing](https://github.com/NLeSC/guide/pull/296)
- [x] Afterwards: check for broken links (globally! the PR link checker only checks changed files...)

**Comment by egpbos** [comment]
Because of the NLeSC-internal reference, this should wait until #394 is merged.

**Comment by egpbos** [comment]
#394 is merged, this issue is unblocked.

**Comment by egpbos** [comment]
This should actually be merged into licensing, will add to #401...
